src/components/CodeEditor.tsx

```diff
@@ -133,7 +133,7 @@ const CodeEditor = ({ roomId }: { roomId: string }) => {
 
   return (
     <div className="h-[calc(100vh-3.5rem)] flex flex-col">
-      <div className="border-b p-2 bg-card">
+      <div className="border-b p-2 bg-card ">
```

Comment:
```suggestion
      <div className="border-b p-2 bg-card">
```
Please fix formatting

Reply:
Please remove the space end of classname